gflib/window.c

```diff
@@ -712,3 +712,13 @@ static u32 GetNumActiveWindowsOnBg8Bit(u32 bgId)
     }
     return windowsNum;
 }
+
+u32 WindowWidthPx(u32 windowId)
+{
+    return gWindows[windowId].window.width * TILE_WIDTH;
+}
```

Comment:
Is there a risk of OOB here?

Reply:
Yeah, if you pass an invalid `windowId` then you'll read OOB. I'm not particularly worried about people doing that, because whatever code is asking for the width is about to use that same `windowId` to draw text. But ig we could put in a `if (windowId >= MAX_WINDOWS) return ?;`, although I'm not sure exactly what we'd want to return. `0`?

Reply:
Fair, I wouldn't know what to return either.